### PR TITLE
Add Ipay Options

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

File: spec/definitions/Gateways/AcquirerName.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ enum:
   - Alipay
   - AIB
   - AstroPay Card
+  - IPay Options
```
Comment (Zaphell) [inline spec/definitions/Gateways/AcquirerName.yaml]: `Ipay Options` according to their website...

File: spec/definitions/GatewayAccountConfig/IPayOptions.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      gatewayConfig:
+        type: object
+        description: IPayOptions credentials object
+        properties:
+          sid:
+            type: string
+            description: IPayOptions webisite ID
+          rcode:
+            type: string
+            description: IPayOptions rcode
```
Comment (Zaphell) [inline spec/definitions/GatewayAccountConfig/IPayOptions.yaml]: `Ipay Options`

File: spec/definitions/Gateways/GatewayName.yaml [hunk 3]
```diff
@@ -25,6 +25,7 @@ enum:
  - iCheque
  - Ilixium
  - Intuit
+ - IPayOptions
```
Comment (Zaphell) [inline spec/definitions/Gateways/GatewayName.yaml]: `IpayOptions` here I guess?

File: spec/definitions/GatewayAccountConfig/IpayOptions.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    required:
+     - "gatewayConfig"
+    properties:
+      gatewayConfig:
+        type: object
+        description: Ipay Options credentials object
+        properties:
+          sid:
+            type: string
+            description: Ipay Options webisite ID
```
Comment (adamaltman) [inline spec/definitions/GatewayAccountConfig/IpayOptions.yaml]: typo

**Pull request merged**